Apps/Sandcastle/CesiumSandcastle.js

```diff
@@ -957,11 +958,13 @@ require({
                 }
             }
         };
-        return loadWithXhr({
+
+        var resource = new Resource({
             url : 'https://api.github.com/gists',
             data : JSON.stringify(data),
             method : 'POST'
```

Comment:
Does `method` still need to be part of the resource?  Wouldn't this just be controlled by the function you are calling?  For example, it seems weird to call `fetch` below, why not just have a `post` function on Resource?